fuzz.go

```diff
@@ -146,20 +146,41 @@ func (f *Fuzzer) Fuzz(obj interface{}) {
 		panic("needed ptr!")
 	}
 	v = v.Elem()
-	f.doFuzz(v)
+	f.doFuzz(v, 0)
 }
 
-func (f *Fuzzer) doFuzz(v reflect.Value) {
-	if !v.CanSet() {
-		return
+// FuzzNoCustom is just like Fuzz, except that any custom fuzz function for
+// obj's type will not be called.  This applies only to obj and not other
+// instances of obj's type.
+// Not safe for cyclic or tree-like structs!
+// obj must be a pointer. Only exported (public) fields can be set (thanks, golang :/ )
+// Intended for tests, so will panic on bad input or unimplemented fields.
+func (f *Fuzzer) FuzzNoCustom(obj interface{}) {
```

Comment:
It's totally your call.  I like that it started with Fuzz, but I'll go with whatever you like :)